src/tests/multihost/alltests/test_sssctl_ldap.py

```diff
@@ -20,6 +20,20 @@ def client_version(multihost):
         return True
 
 
+def configure_proxy(multihost):
```

Comment:
Can we add this in common library, so anyone can use it?

Reply:
This is not common library , this can be only used for this script

Reply:
Yes, I knew its not from common library, I was saying can we add it in common lib so anyone can reuse.